src/glue/jobs/s3_to_json_s3.py

```diff
@@ -167,32 +167,36 @@ def get_json_schema(archive_map, file_metadata, json_schemas):
             "archive_map_version": os.environ.get("archive_map_version")
     }
     file_metadata["assessment_revision"] = int(file_metadata["assessment_revision"])
+    valid_assessments = []
     for assessment in archive_map["assessments"]:
         if (assessment["assessmentIdentifier"] == file_metadata["assessment_id"]
-                and assessment["assessmentRevision"] == file_metadata["assessment_revision"]):
-            for file in assessment["files"]:
-                if file["filename"] == file_metadata["file_name"]:
-                    json_schema["url"] = file["jsonSchema"]
-                    json_schema["schema"] = _get_cached_json_schema(
-                            url=json_schema["url"],
-                            json_schemas=json_schemas
-                    )
-                    return json_schema
+                and assessment["assessmentRevision"] <= file_metadata["assessment_revision"]):
+            valid_assessments.append(assessment)
+    if len(valid_assessments):
```

Comment:
Nit: If you're going to say `len` then I'd prefer you have `len(my_list) > 0`, otherwise, you can just do `if valid_assessments:`

Reply:
I'll lint.